### DISCUSSION TITLE
[Feature Request] Add ability to change HDD model label

### DISCUSSION THREAD

**Discussion by mw308** [86Box/86Box#4243] [Ideas]
Please could you add an additional box to the HDD settings so we can set our own model label instead of it being 86B_HD00 etc?  Similar to how the CDROMs now have model names and revisions, but a free-type text box so we can put in "ST3211A" for example.

Thanks

**Comment by Fubuki0x10DE** [comment]
This is not easy because it may not compatible with Seagate's DOS software. But emulate CompatFlash as HDD can be considered.

**Comment by mw308** [comment]
I'm not sure where Seagate's DOS software comes into it? The label could be any HDD manufacturer, Maxtor, IBM etc.  I only used a Seagate model number as an example.  I also don't mean to emulate other disk types, only to change the label reported to the OS instead of it showing 86B_HDxx.

**Comment by Fubuki0x10DE** [comment]
> I'm not sure where Seagate's DOS software comes into it? The label could be any HDD manufacturer, Maxtor, IBM etc. I only used a Seagate model number as an example. I also don't mean to emulate other disk types, only to change the label reported to the OS instead of it showing 86B_HDxx.

For example, SeaTools, MAXLLF.

**Comment by mw308** [comment]
I understand _what_ the software is, but I'm not asking about that.  I'm asking about the label presented to the BIOS and therefore shown in POST, before any OS loads, which may not even be DOS.
